.github/workflows/.editorconfig

```diff
@@ -0,0 +1,2 @@
+[*.yml]
+indent_size = 2
```

Comment:
Is this usually committed? If it's only to control tabs when modifying .yml files in this directory, it feels kinda overkill

Reply:
Yes and no.. 

There should be an .editorconfig in root too. Then this one will make (more) sense.